tendrl/gluster_integration/manager/__init__.py

```diff
@@ -34,10 +34,12 @@ def main():
     NS.state_sync_thread = sds_sync.GlusterIntegrationSdsSyncStateThread()
 
     NS.node_context.save()
-    NS.tendrl_context.save()
-    NS.gluster_integration.definitions.save()
-    NS.gluster_integration.config.save()
+    NS.gluster.tendrl_context.save()
```

Comment:
No tendrl_context required in alerting and performance_monitoring applications currently(after integration id is made optional)....